src/v/model/tests/randoms.h

```diff
@@ -123,16 +127,29 @@ inline model::broker random_broker(model::node_id node_id) {
       std::move(kafka_advertised_listeners),
       tests::random_net_address(),
       rack,
-      random_broker_properties(),
+      broker_properties,
     };
 }
 
+inline model::broker random_broker(model::node_id node_id) {
```

Comment:
This can be removed and replaced by a the 2-arg overload above with a `= random_broker_properties()` default arg.